git_hooks/README.md

```diff
@@ -0,0 +1,10 @@
+README for git hooks script
+===========================
+git has a way to run scripts, which are invoked by specific git commands.
+The git hooks are located in <root>.git/hooks , and as such are not under version control
+for more information, see https://git-scm.com/docs/githooks
+
+The mbedTLS git hooks are located in <root>/git_hooks directory, and one must create a soft link from <root>.git/hooks to <root>/git_hooks , in order to make the hook scripts successfully work.
+example:
+
+.git/hooks> ln -s ../../git_hooks/pre-push pre-push
```

Comment:
I tried using git bash on windows, and it actually created a copy rather than a link.